www/base/src/app/app.module.js

```diff
@@ -13,3 +21,8 @@ class App {
 
 
 angular.module('app', new App());
+
+require('./common/common.module.js');
+
+const context = require.context('./', true, /^(?!.*(?:module|spec|webpack.js$)).*\.js$/);
```

Comment:
I think this is better to keep it explicit.

I know this trick is import(*.js) but it is really not obvious.

So I would just import stuff line by line

Reply:
Agreed.

Reply:
Fixed now.